### PR TITLE
Add SECURITY.md, update CONTRIBUTING.md, surface in README.md

### PR DESCRIPTION
**Pull request by totalamateurhour**
- Update CONTRIBUTING.md with new OCA information
- Add SECURITY.md for information on how to flag security issues
- Update README.md with the above

Signed-off-by: Sergio Leunissen <sergio.leunissen@oracle.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,15 +33,23 @@ can be accepted.
...
+1. We will assign the pull request to 2-3 people for review before it is merged.
 
-Copyright (c) 2018, 2020 Oracle and/or its affiliates.
+## Code of conduct
+
+Follow the [Golden Rule](https://en.wikipedia.org/wiki/Golden_Rule). If you'd
+like more specific guidelines, see the [Contributor Covenant Code of Conduct][COC].
+
+[OCA]: https://oca.opensource.oracle.com
+[COC]: https://www.contributor-covenant.org/version/1/4/code-of-conduct/
```
Comment (AmedeeBulle) [inline CONTRIBUTING.md:55]: We might want to use the latest version of the COC

```suggestion
[COC]: https://www.contributor-covenant.org/version/2/0/code_of_conduct/
```

**Review by AmedeeBulle — COMMENTED** [review]
LGTM -- Need to check the version of the COC.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,29 +1,30 @@
...
 
-All pull requests require the contributor to have agreed to and signed the
-[Oracle Contributor Agreement](https://www.oracle.com/technetwork/community/oca-486395.html) (OCA).
+## Contributing code
 
-For pull requests to be accepted, the bottom of your commit message must have
-the following line using your name and e-mail address as it appears in the
-OCA signatories list.
+We welcome your code contributions. Before submitting code via a pull request,
+you will need to haved signed the [Oracle Contributor Agreement][OCA] (OCA) and
```
Comment (PaulNeumann) [inline CONTRIBUTING.md:16]: I think "haved" should be "have".

**Review by gvenzl — APPROVED** [review]

**Comment by gvenzl** [comment]
LGTM

**Review by gvenzl — CHANGES_REQUESTED** [review]
Please address the comments from @PaulNeumann.

File: CONTRIBUTING.md (hunk 2 above)
Comment (totalamateurhour) [inline CONTRIBUTING.md:16]:
```suggestion
you will need to have signed the [Oracle Contributor Agreement][OCA] (OCA) and
```

Comment (totalamateurhour) [reply CONTRIBUTING.md]: Thanks, I made this change.

**Pull request merged**